icons/heart.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M20.42 4.58a5.4 5.4 0 0 0-7.65 0l-.77.78-.77-.78a5.4 5.4 0 0 0-7.65 0C1.46 6.7 1.33 10.28 4 13l8 8 8-8c2.67-2.72 2.54-6.3.42-8.42z" />
+  <path d="M20.4,4.6c-2.1-2.1-5.6-2.1-7.8,0L12,5.2l-0.6-0.6c-2.1-2.1-5.6-2.1-7.8,0c-2.1,2.1-2.1,5.6,0,7.7L12,21 l8.4-8.7C22.5,10.2,22.5,6.7,20.4,4.6z" />
```

Comment:
Yeah, I can totally see where you're coming from, only, I think this should be our base shape instead (it's more pixel perfect and also has better curvature):
```suggestion
  <path d="M19.11 14.06 12 21l-7.11-6.94C3.39 12.61 2 10.8 2 8.5a5.5 5.5 0 0 1 10-3.16A5.5 5.5 0 0 1 22 8.5c0 2.29-1.4 4.1-2.89 5.56Z" />
```

Reply:
How are you defining “pixel perfect” here? I’ve been mostly focused on “grid perfect” (‘snap to grid’ in Adobe Illustrator)…

Reply:
Well, pixel perfection is mostly about sharpness at 100% view, horizontal, vertical and diagonal lines are the sharpest if the lines stick _exactly_ on the pixel grid, which our current heart icon does not.

On the other hand, funky dimensions are usually the result of circle arcs being handled differently by editors, e.g. we have this shape:
![image](https://user-images.githubusercontent.com/17746067/230713990-b4aaf3b7-0ce8-44e1-933f-e0758b5801c2.png)
As @jguddas's amazing tool illustrates, the top part consists of two single circle arcs.

Unfortunately neither Figma nor Illustrator support circle arcs and represent them with the next best thing they have, which is cubic curves, so these are their respective representations of the shape above:

Figma adds a bunch of control points and renders the shape at 20 by 18 pixels:
![image](https://user-images.githubusercontent.com/17746067/230714113-9b2792e7-56ff-4fd1-847a-f28ae05a37dc.png)

Illustrator adds just enough control points because of which the path ends up being assymetric, and Illustrator renders it at 20 by 17.9977 pixels:
![image](https://user-images.githubusercontent.com/17746067/230714135-23956ef8-330e-43bd-b1fb-1f1e6cb4573b.png)

Reply:
I see. Yeah Iv’e noticed Illustrator sub-optimally handling certain SVG elements… :/